environment.yml

```diff
@@ -9,59 +9,73 @@ dependencies:
   - python-dateutil>=2.5.0
   - pytz
 
-  # development
+  # benchmarks
   - asv
+
+  # building
   - cython>=0.28.2
+
+  # code checks
   - flake8
-  - flake8-comprehensions
-  - flake8-rst>=0.6.0,<=0.7.0
-  - gitpython
-  - hypothesis>=3.82
-  - ipywidgets
-  - isort
-  - moto
+  - flake8-comprehensions  # used by flake8, linting of unnecessary comprehensions
+  - flake8-rst>=0.6.0,<=0.7.0  # linting of code blocks in rst files
+  - isort  # check that imports are in the right order
   - mypy
+  - pycodestyle  # used by flake8
+  - pip:
+    - cpplint
+
+  # documentation
+  - gitpython  # obtain contributors from git for whatsnew
+  - sphinx
+  - numpydoc>=0.9.0
+
+  # documentation (jupyter notebooks)
   - nbconvert>=5.4.1
-  - nbformat
-  - notebook>=5.7.5
+  - nbsphinx
   - pandoc
-  - pycodestyle
-  - pyqt
-  - python-snappy
+
+  # testing
+  - boto3
+  - botocore>=1.11
+  - hypothesis>=3.82
+  - moto  # mock S3
   - pytest>=4.0.2
+  - pytest-cov
   - pytest-mock
-  - sphinx
-  - numpydoc>=0.9.0
+  - pytest-xdist
+  - seaborn
+  - statsmodels
+
+  # unused (required indirectly may be?)
+  - ipywidgets
+  - nbformat
+  - notebook>=5.7.5
+  - python-snappy
```

Comment:
This would be for parquet. In the past there was an issue with either pyarrow or fastpaquet essentially requiring snappy compression (I don't recall the details).